lib/middleware/websocket/controller.rb

```diff
@@ -11,13 +12,17 @@ def initialize
 
     def on_open(connection)
       generate_client(connection: connection)
-
-      Interactor::UpdateAllClients.new.call(clients: @clients)
+      Interactor::UpdateClients.new.race_update(clients: @clients)
     end
 
     def on_message(connection, data)
-      #update the Client in question
-      #update_all_clients
+      data = JSON.parse(data)
+      client = find_client(connection: connection).first
+
+      if (data['token'] == client.player.token)
```

Comment:
We may not need to use the token if we're only using web sockets. If the user only connects to the application once, it appears that we'll have a unique reference to them without a token.

Reply:
@jgayfer That's actually true, the `connection` passed in as a parameter is unique, I can use that. I'll opt to keep the UUID token stored as an attribute in each `Client` object since we do generate it in the database and it might be useful later, but I'll move to use the `connection` object to verify connections rather than the UUID token.

Reply:
@jgayfer fixed.